src/lib.rs

```diff
@@ -61,7 +61,9 @@ pub struct Response {
     /// An error if there is one, or null
     pub error: Option<error::RpcError>,
     /// Identifier from the request
-    pub id: json::Value
+    pub id: json::Value,
+    /// jsonrpc field, MUST be "2.0"
+    pub jsonrpc: json::Value
```

Comment:
yeah, sure, should I make it optional for the request to then?

I think it might be a better idea to separate the two implementations though, thats at least what I have found to be common when working with jsonrpc implementations.